src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -5555,23 +5570,37 @@ static void Main()
 
             var discard1 = GetDiscardDesignations(tree).First();
             Assert.Null(model.GetDeclaredSymbol(discard1));
+            Assert.True(model.GetSymbolInfo(discard1).IsEmpty);
             var declaration1 = (DeclarationExpressionSyntax)discard1.Parent;
             Assert.Equal("var _", declaration1.ToString());
-            //Assert.Equal("", model.GetTypeInfo(declaration1).Type.ToTestDisplayString()); //  https://github.com/dotnet/roslyn/issues/15450
+            Assert.Equal("System.Int64", model.GetTypeInfo(declaration1).Type.ToTestDisplayString());
```

Comment:
Filed [https://github.com/dotnet/roslyn/issues/16316](https://github.com/dotnet/roslyn/issues/16316)

---
In reply to: [95042876](https://github.com/dotnet/roslyn/pull/16153#discussion_r95042876) [](ancestors = 95042876,95025399)